osv/models.py

```diff
@@ -667,13 +667,24 @@ def to_vulnerability(self, include_source=False):
         cr.type = vulnerability_pb2.Credit.Type.Value(credit.type)
       credits_.append(cr)
 
+    related_bugs = Bug.query(Bug.related == self.db_id).fetch()
+    related_bug_ids = [bug.db_id for bug in related_bugs]
+    related = sorted(list(set(related_bug_ids + self.related)))
+
+    alias_group = AliasGroup.query(AliasGroup.bug_ids == self.db_id).get()
+    aliases = self.aliases
```

Comment:
I think we should use an empty list if there's no AliasGroup, in case it declares too many or we have it on the denylist.